app/views/incident_reports/_form.html.haml

```diff
@@ -383,9 +383,9 @@
       .field
         = form.label :reason_not_up_to_curb
         = form.text_field :reason_not_up_to_curb, id: :incident_report_reason_not_up_to_curb
-      .field
-        = form.label :vehicle_in_bus_stop_plate
-        = form.text_field :vehicle_in_bus_stop_plate, id: :incident_report_vehicle_in_bus_stop_plate
+    .field
```

Comment:
Might just be GitHub, but it looks like this indentation is offset by an odd number of spaces.

Reply:
good catch.